### ISSUE TITLE
iOS Crash MGLMapView.mm line 554

### ISSUE THREAD

**Issue by picciano** [mapbox/mapbox-gl-native#1186]
Version.txt is reporting version 9dae9cf06e
Unfortunately, I cannot detect a specific action causing this.

![screen shot 2015-04-02 at 1 57 26 pm](https://cloud.githubusercontent.com/assets/417933/6972352/731d1410-d940-11e4-891d-d783c6214d6f.png)

**Comment by robipresotto** [comment]
i got it everyday too

```ruby
Thread : Crashed: com.apple.main-thread
0  Embark                         0x00000001001c4830 mbgl::Map::renderSync() (memory:2603)
1  Embark                         0x000000010023b140 -[MGLMapView glkView:drawInRect:] (MGLMapView.mm:508)
2  Embark                         0x000000010023b140 -[MGLMapView glkView:drawInRect:] (MGLMapView.mm:508)
3  GLKit                          0x000000018328b3a4 -[GLKView _display:] + 364
4  QuartzCore                     0x000000018638ad88 CA::Layer::display() + 148
5  QuartzCore                     0x000000018636c570 CA::Layer::display_if_needed(CA::Transaction*) + 244
6  QuartzCore                     0x000000018636c228 CA::Layer::layout_and_display_if_needed(CA::Transaction*) + 44
7  QuartzCore                     0x000000018636ba14 CA::Context::commit_transaction(CA::Transaction*) + 276
8  QuartzCore                     0x000000018636b798 CA::Transaction::commit() + 436
9  UIKit                          0x0000000186a14478 _afterCACommitHandler + 156
10 CoreFoundation                 0x00000001821cad98 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 32
11 CoreFoundation                 0x00000001821c7d24 __CFRunLoopDoObservers + 360
12 CoreFoundation                 0x00000001821c8104 __CFRunLoopRun + 836
13 CoreFoundation                 0x00000001820f51f4 CFRunLoopRunSpecific + 396
14 GraphicsServices               0x000000018b43f6fc GSEventRunModal + 168
15 UIKit                          0x0000000186a8610c UIApplicationMain + 1488
16 Embark                         0x0000000100046474 main (main.m:14)
17 libdyld.dylib                  0x0000000193a72a08 start + 4
````

**Comment by 1ec5** [comment]
Fixed in #1254.
